active_model_serializers.gemspec

```diff
@@ -43,6 +43,7 @@ Gem::Specification.new do |spec|
   # 'thread_safe'
 
   spec.add_runtime_dependency 'jsonapi', '0.1.1.beta2'
+  spec.add_runtime_dependency 'case_transform', '>= 0.2'
```

Comment:
I'm cool with that. 
should we email github to just assign the existing repo to rails-api?